### PR TITLE
Show contributor avatar in headline for comment articles

### PR DESCRIPTION
**Pull request by oliverlloyd**
## What does this change?
Support the author avatar showing below the headline for Comment pieces

Screenshot 2020-02-03 at 10 16 23

### Changes of note
To make this work I've used flex for positioning and a min height of `259px` to achieve the correct spacing. This css in frontend if very different but the end results match and hopefully this code is simpler.

## Why?
Because it looks great there

## Link to supporting Trello card
https://trello.com/c/eU8BeKbM/1049-author-image-shown-in-article-header

### CODE REVIEW

File: src/web/layouts/CommentLayout.tsx [hunk 1]
```diff
@@ -150,6 +147,32 @@ const maxWidth = css`
     }
 `;
 
+const avatarHeadlineWrapper = (hasAvatar: boolean) => css`
+    min-height: ${hasAvatar && '259px'};
```
Comment (liywjl) [inline src/web/layouts/CommentLayout.tsx]: I would prefer: `${hasAvatar ? 'min-height: 259px' : ''};`

File: src/web/layouts/CommentLayout.tsx [hunk 2]
```diff
@@ -235,25 +261,42 @@ export const CommentLayout = ({ CAPI, NAV }: Props) => {
...
-                                <ArticleHeadline
-                                    headlineString={CAPI.headline}
-                                    designType={CAPI.designType}
-                                    pillar={CAPI.pillar}
-                                    webPublicationDate={CAPI.webPublicationDate}
-                                    tags={CAPI.tags}
-                                    byline={CAPI.author.byline}
-                                />
-                            </ArticleHeadlinePadding>
+                            <div className={avatarHeadlineWrapper(!!avatarUrl)}>
```
Comment (liywjl) [inline src/web/layouts/CommentLayout.tsx]: take a step back why not use `cx(baseStyles, hasAvatar ? minHeightStyles : '')` ?

Comment (oliverlloyd) [reply src/web/layouts/CommentLayout.tsx]: That's a really good suggestion, this makes the code much more readable.

I stuck with the && pattern as that's the pattern used elsewhere but 👍 for seeing this

File: src/web/layouts/CommentLayout/CommentLayout.tsx [hunk 3]
```diff
@@ -32,8 +32,8 @@ import { parse } from '@frontend/lib/slot-machine-flags';
 
 import GE2019 from '@frontend/static/badges/general-election-2019.svg';
 
-import { Border } from './Border';
-import { GridItem } from './GridItem';
+import { Border } from '../Border';
+import { GridItem } from '../GridItem';
```
Comment (liywjl) [inline src/web/layouts/CommentLayout/CommentLayout.tsx]: Maybe worth  using absolute path

**Pull request merged**